src/wallet/rpcwallet.cpp

```diff
@@ -31,7 +31,14 @@
 
 CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest& request)
 {
-    // TODO: Some way to access secondary wallets
+    if (!request.wallet.empty()) {
+        for (const auto& wallet : ::vpwallets) {
+            if (request.wallet == wallet->GetName()) {
+                return wallet;
+            }
+        }
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Requested wallet does not exist or is not loaded");
```

Comment:
Mixed feelings here, could we just return `nullptr` because if no wallet is loaded then it is the same outcome: wallet not found?